crates/mun_language_server/src/project_manifest.rs

```diff
@@ -0,0 +1,62 @@
+use anyhow::{bail, Result};
```

Comment:
I'm generally not a fan of shadowing `std` types that are in the `std` prelude, as it's impossible for anyone reading the code to know

Reply:
Agree, fixed.